testutil/integration/validators.go

```diff
@@ -17,7 +26,7 @@ func CreateValidators(n int) (
 		signersByAddress = make(map[string]tmtypes.PrivValidator, n)
 	)
 	for i := 0; i < n; i++ {
-		privVal := mock.NewPV()
+		privVal := mock.PV{PrivKey: ed25519.GenPrivKeyFromSecret([]byte(chainId + fmt.Sprint(i)))}
```

Comment:
Changed to make this deterministic. This caused some other unrelated files to need to change.